### PR TITLE
DEL: Remove package

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/fp6z7w35ugkis5mauu6i7iqfie?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1863)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ew4zxguj6pflx5ayqsa2ph65qq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1864)

File: hooks/useBiometrics.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import { useState, useEffect } from 'react';
+import { useState, useEffect, useCallback } from 'react';
 import { Alert, Platform } from 'react-native';
 import ReactNativeBiometrics, { BiometryTypes as RNBiometryTypes } from 'react-native-biometrics';
```
Comment (Overtorment) [inline hooks/useBiometrics.ts:3]: does it make sense to fork [react-native-biometrics](https://github.com/SelfLender/react-native-biometrics) under our org?

**Review by Overtorment — APPROVED** [review]
fuk yes

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**